MixpanelDemo/MixpanelDemo/TrackingViewController.swift

```diff
@@ -45,11 +46,12 @@ class TrackingViewController: UIViewController, UITableViewDelegate, UITableView
 
         switch indexPath.item {
         case 0:
-            let ev = "Track Event!"
+            eventNo += 1
+            let ev = "Track Event! No: \(eventNo)"
             Mixpanel.mainInstance().track(event: ev)
             descStr = "Event: \"\(ev)\""
         case 1:
-            let ev = "Track Event With Properties!"
+            let ev = "Track Event With Properties! No: \(eventNo)"
```

Comment:
in this example, I would make the event number a property and keep the event name consistent

Reply:
oops, thx for catching this